src/connection.h

```diff
@@ -333,6 +365,28 @@ static inline int connAddrSockName(connection *conn, char *ip, size_t ip_len, in
     return connAddr(conn, ip, ip_len, port, 0);
 }
 
+/* Format the connection address and store the result in daddr/saddr, except Unix socket */
+static inline int connFmtName(connection *conn) {
+    char addr[NET_ADDR_STR_LEN];
+    int ret;
+
+    ret = connFormatAddr(conn, addr, sizeof(addr), 0);
+    if (ret) {
+        conn->daddr = strdup(addr);
+    } else {
+        conn->daddr = NULL;
+    }
+
+    ret = connFormatAddr(conn, addr, sizeof(addr), 1);
+    if (ret) {
+        conn->saddr = strdup(addr);
+    } else {
+        conn->saddr = NULL;
+    }
+
+    return 0;
+}
```

Comment:
I think we should populate these two fields as part of the accept handler and remove this one along with `connFormatAddr`. everyone should now use `conn->saddr` and `conn-daddr`.